docusaurus/docs/cloud/advanced/email.md

```diff
@@ -0,0 +1,132 @@
+
+:::prerequisites
+
+- A local Strapi project running on `v4.8.2+`.
+- Credentials for a third-party email provider (see [Strapi Market](https://market.strapi.io/)).
+
+:::
+
+## Configuration
+To configure a third-party email provider, in your Strapi project, create a `./config/env/production/plugins.js` or `./config/env/production/plugins.ts` file with the following content:
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  Required Email Provider Plugin Configuration
+  // ...
+});
+```
+Each provider will have different configuration settings available. Review the respective entry for that provider in the [Marketplace](https://market.strapi.io/).
+
+Below are example configurations for the Email plugins.
+
+<Tabs groupId="email-examples" >
+
+<TabItem value="sendgrid" label="Sendgrid">
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  email: {
+    config: {
+      provider: 'sendgrid',
+      providerOptions: {
+        apiKey: env('SENDGRID_API_KEY'),
+      },
+      settings: {
+        defaultFrom: 'myemail@protonmail.com',
+        defaultReplyTo: 'myemail@protonmail.com',
+      },
+    },
+  },
+  // ...
+});
+```
+</TabItem >
+<TabItem value="amazon-ses" label="Amazon SES">
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  email: {
+    config: {
+      provider: 'amazon-ses',
+      providerOptions: {
+        key: env('AWS_SES_KEY'),
+        secret: env('AWS_SES_SECRET'),
+        amazon: 'https://email.us-east-1.amazonaws.com',
+      },
+      settings: {
+        defaultFrom: 'myemail@protonmail.com',
+        defaultReplyTo: 'myemail@protonmail.com',
+      },
+    },
+  },
+  // ...
+});
+```
+</TabItem>
+<TabItem value="mailgun" label="Mailgun">
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  email: {
+    config: {
+      provider: 'mailgun',
+      providerOptions: {
+        key: env('MAILGUN_API_KEY'), // Required
+        domain: env('MAILGUN_DOMAIN'), // Required
+        url: env('MAILGUN_URL', 'https://api.mailgun.net'), //Optional. If domain region is Europe use 'https://api.eu.mailgun.net'
+      },
+      settings: {
+        defaultFrom: 'myemail@protonmail.com',
+        defaultReplyTo: 'myemail@protonmail.com',
+      },
+    },
+  },
+  // ...
+});
+```
+</TabItem>
+</Tabs>
+
+:::caution
+The email provider must be installed as a package dependency in your Strapi project. For example, to use the Sendgrid email provider, install the `@strapi/provider-email-sendgrid` package.
+:::
+
+## Strapi Cloud Configuration
+Before pushing changes to GitHub, add environment variables to the Strapi Cloud project:
+
+1.  Log into Strapi Cloud and click on the corresponding project on the Projects page.
+2.  Click on the **Settings** tab and choose **Variables** in the left menu.
+3.  Add the required environment variables specific to the email provider.
+4.  Click **Save**.
+
+### Mailgun Example:
```

Comment:
```suggestion
**Example:**
```
No need for a title here. Also, question: why is there only a MailGun example here when in the previous section there were examples of Sendgrid and Amazon SES as well? 😊